
# Filename for user Python script containing task class
string user_script_filename

# Name of class that inherits from either optas_ros.optas_ros.Controller or optas_ros.optas_ros.Planner
string task_cls_name

---

bool success
string message